addons/website/static/src/js/tours/tour_utils.js

```diff
@@ -348,6 +348,26 @@ export function clickOnText(snippet, element, position = "bottom") {
     ];
 }
 
+/**
+ * Wait for a specific amount of time.
+ * @param {number} delay
+ * @returns {TourStep[]}
+ */
+export function delayNextStep(delay) {
```

Comment:
This would definitely not be ok:
1. Just the fact of having the wait for a *delay* in a tour is not ok
2. The tour should test the strictest flow anyway: which is clicking immediately (not after hundreds of milliseconds)